Gemfile.lock

```diff
@@ -138,7 +141,7 @@ GEM
     crack (0.4.3)
       safe_yaml (~> 1.0.0)
     crass (1.0.3)
-    cucumber (2.4.0)
+    cucumber (2.99.0)
```

Comment:
I don't think all those bumps are necessary for your PR, are they?

Reply:
Updating that many dependencies as drive-by doesn't look like a good idea to me, but it can be that some of them got updated because we didn't update many gems for a long time now. I planned to do a gem-update PR again soon and it will probably create a lot of conflicts in the Gemfile.lock, but you can rebase after that and generate a new Gemfile.lock with your changes, then it should look a lot cleaner.

Reply:
Agreed!

Reply:
By the way, the gems were updated in #7772 which was merged 4 days ago and which is the reason you have now many conflicts in the Gemfile and Gemfile.lock. But you can just rebase and the latest `develop` branch and you should have a much cleaner Gemfile.lock then :)